curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.md

```diff
@@ -23,7 +23,7 @@ myTest();
 console.log(loc);
 ```
 
-The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line will throw an error, as `loc` is not defined outside of the function.
+The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line (outside the myTest function) will throw an error, as `loc` is not defined outside of the function.
```

Comment:
```suggestion
The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line (outside of the `myTest` function) will throw an error, as `loc` is not defined outside of the function.
```
A bit of nitpicking. I wondered also, if simply referring to this `console.log(loc)` as a _second_ one wouldn't work.

---

Note for future - it is recommended to make changes on a separate branch, than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally.

Reply:
Sorry, I am new to open source. And thanks for pointing out my mistake.